app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -203,6 +205,9 @@ abstract class AppDatabase : RoomDatabase() {
       teleconsultFacilityWithMedicalOfficersDao().clear()
       teleconsultRecordDao().clear()
       callResultDao().clear()
+
+      //Todo Sid - Do we need to clear the questionnaire table
+      questionnaireDao().clearData()
```

Comment:
We use `clearAppData` in `SyncAndClearPatientData` class. If we are gonna clear the questionnaire data, we also need to delete the pull token there.